js/ui/codeView.js

```diff
@@ -1470,11 +1474,14 @@ var CodeViewManager = GObject.registerClass({
         if (!appInfo)
             return false;
 
+        let gtkId = actor.meta_window.get_gtk_application_id();
+
         // The custom X-Endless-Hackable key has the last word always
         if (appInfo.has_key(_HACKABLE_DESKTOP_KEY)) {
             if (!appInfo.get_boolean(_HACKABLE_DESKTOP_KEY))
                 return false;
-        } else {
+        // The hack unlock has the com.endlessm.Clubhouse id in the katamari flatpak
+        } else if (gtkId !== 'com.endlessm.HackUnlock' && gtkId !== 'com.endlessm.HackToolbox') {
```

Comment:
I think the string here is correct because you are checking the app ID, not the namespace of the flatpak, but I am a bit confused by the comment.

Reply:
I've tried to explain better in the comment:

```
        // HackUnlock and HackToolbox are inside the com.endlessm.Clubhouse flatpak
        // and have the Clubhouse appInfo, so we should ignore those cases here
        // to be able to show the HackUnlock and HackToolbox windows
```